Formula/parmetis.rb

```diff
@@ -0,0 +1,72 @@
+class Parmetis < Formula
+  desc "MPI library for graph/mesh partitioning and fill-reducing orderings"
+  homepage "http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview"
+  url "http://glaros.dtc.umn.edu/gkhome/fetch/sw/parmetis/parmetis-4.0.3.tar.gz"
+  sha256 "f2d9a231b7cf97f1fee6e8c9663113ebf6c240d407d3c118c55b3633d6be6e5f"
+
+  # METIS 5.* is required. It comes bundled with ParMETIS.
+  # We prefer to brew it ourselves.
```

Comment:
You can simplify these two lines into something like `# Prefer our metis over the bundled one`.